src/main/java/org/veupathdb/service/eda/ss/model/db/DB.java

```diff
@@ -68,6 +68,8 @@ interface Columns extends ColumnCollection {
         String UNITS_COL_NAME = "unit";
         String PRECISION_COL_NAME = "precision";
         String SCALE_COL_NAME = "scale";
+        String HAS_STUDY_DEPENDENT_VOCABULARY = "has_study_dependent_vocabulary";
+        String VARIABLE_SPEC_TO_IMPUTE_ZEROES_FOR = "variable_spec_to_impute_zeroes_for";
```

Comment:
I see, so this is the DB col name.  I assume this is a variable ID then; I still think we should change to `variable_id_to_impute_zeroes_for` to avoid confusion.  Can keep it that way all the way to the metadata API or convert to a VariableSpec at some point and change the name back.